modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -1116,6 +1139,13 @@ void RenderableFieldlinesSequence::render(const RenderData& data, RendererTasks&
         }
 
         glBindVertexArray(_vertexArrayObject);
+       /* #ifdef __APPLE__
```

Comment:
is this a comments out ifdef? unclear

Reply:
ops, forgot about this. Copy/paste mistake